package.json

```diff
@@ -51,38 +66,40 @@
   },
   "homepage": "https://github.com/ERC725Alliance/erc725.js#readme",
   "devDependencies": {
-    "@types/chai": "^4.3.4",
-    "@types/isomorphic-fetch": "^0.0.36",
-    "@types/jest": "^27.5.2",
-    "@types/mocha": "^10.0.1",
-    "@types/node": "^18.11.19",
-    "@types/sinon": "^10.0.13",
-    "@typescript-eslint/eslint-plugin": "^5.50.0",
-    "@typescript-eslint/parser": "^5.50.0",
-    "assert": "^2.0.0",
-    "chai": "^4.3.7",
-    "eslint": "^8.33.0",
+    "@types/chai": "~4.3.11",
+    "@types/jest": "~27.5.2",
+    "@types/mocha": "~10.0.6",
+    "@types/node": "^20.11.16",
```

Comment:
I think it makes even more sense to use node v20 as default in `.tools-version`, since we are using the types for v20 here

Reply:
Types are always backward compatible, but I switched to v20, since you're right v18 is now old school :)